src/draw/vg_lite/lv_draw_vg_lite_arc.c

```diff
@@ -180,6 +180,9 @@ void lv_draw_vg_lite_arc(lv_draw_unit_t * draw_unit, const lv_draw_arc_dsc_t * d
             vg_lite_identity(&path_matrix);
             lv_vg_lite_matrix_multiply(&path_matrix, &u->global_matrix);
 
+            /* move image to center */
+            vg_lite_translate(cx - radius_out, cy - radius_out, &matrix);
```

Comment:
What's the behavior of software renderer when image is smaller than arc widget size?